### ISSUE TITLE
Oderengesall: worth hearing related to omding, omchederenges. (anyone heard this?)

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#129]
Oderengesall: worth hearing related to omding, omchederenges. (anyone heard this?) created by mngiruchelbad@gmail.com on 2017-01-24 14:11:00

**Comment by s-o-u-a-n-g** [comment]
Correct. If someone says “Tia Oderngesall” I’d stop what I’m doing and pay attention. It’s usually to get everyone on the same page at a family meeting, or to announce the result of a decision. 

“Listen up, this is important.” Worthy of being heard. 

souang
